contracts/services/forwarder/src/msg.rs

```diff
@@ -0,0 +1,247 @@
+use cosmwasm_schema::{cw_serde, QueryResponses};
+use cosmwasm_std::{Addr, Deps, DepsMut, Uint128};
+use cw_utils::Duration;
+use getset::{Getters, Setters};
+use std::collections::HashMap;
+use valence_macros::OptionalStruct;
+use valence_service_utils::{
+    denoms::{CheckedDenom, DenomError, UncheckedDenom},
+    error::ServiceError,
+    msg::ServiceConfigValidation,
+    ServiceConfigInterface,
+};
+
+#[cw_serde]
+/// Enum representing the different action messages that can be sent.
+pub enum ActionsMsgs {
+    /// Message to forward tokens.
+    Forward {},
+}
+
+#[cw_serde]
+#[derive(QueryResponses)]
+/// Enum representing the different query messages that can be sent.
+pub enum QueryMsg {
+    /// Query to get the owner address.
+    #[returns(Addr)]
+    GetOwner {},
```

Comment:
the cw_ownable crate has a #[cw_ownable_query] macro that will add the ownership query here. I'd suggesting using that instead of redefining it here.